.github/workflows/ci-comment.yml

```diff
@@ -55,4 +55,25 @@ jobs:
               issue_number: issue_number,
               body: `The commit ${pr_sha} (as a parent of ${merge_sha}) contains errors. Please inspect the [Run Summary](${context.payload.workflow_run.html_url}) for details.`
             });
+            await octokit.request(`POST /repos/{owner}/{repo}/issues/{issue_number}/labels`, {
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              issue_number: issue_number,
+              labels: [ 'waiting: response from author' ]
+            })
```

Comment:
Any reason to use the raw POST over [`addLabels`](https://octokit.github.io/rest.js/v18#issues-add-labels) (all PRs are issues)?

Reply:
Laziness.

Reply:
I think we should prefer using the SDK over the rest interface when possible.  It is less likely we'll screw something up, and the code becomes more readable.

Reply:
Agreed here. I'd prefer using a pre-existing action, and will likely do so.